usagov/usagov-2021
# USAGOV-2014-Display-Call-Times: Update callCenterWaitTimeLookup.js

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2014

## Description

On the phone pages (/phone and /es/llamenos) a new alert should show if the contact center is open and the database has recently been refreshed with data. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- You can verify the page is showing correctly by first going to https://s3-us-gov-west-1.amazonaws.com/cg-4d6fb302-315f-403e-a96e-a7563ccddd3d/1.0/waittime.json
- Note the call times in English and Spanish (these are in seconds) 
- On /phone, /es/llamenos, /chat, and /es/chat go to edit and in the body field change the text format to HTML and add `<div data-call-wait='showTime'>&nbsp;</div>` to the end. 
- If it's -1 the alert box should not be visible on the page (call center is closed) 
- Otherwise: 
-- if it's less than a minute ("en":<60) it should always round to a minute. 
-- if it's more than a minute it should round and show the plural minutes or minutos 

- If the value is not -1 but the alert is not showing verify that the timestamp was taken less than 10 minutes ago. See the checkTimeStamp function I included in the script. 

See examples on stage: 
English -> https://cms-stage.usa.gov/phone
Spanish -> https://cms-stage.usa.gov/es/llamenos


## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
